components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -632,27 +642,32 @@
 
                                 if( !("UniqueID".equalsIgnoreCase(propertyName) ) ){
                         %>
-                        <td class="leftCol-med" width="50%" style="text-align:left;" id="<%=name%>"><%=propertyName%>
+                        <td class="leftCol-med" width="50%" style="text-align:left;"
+                        id="<%=Encode.forHtmlAttribute(name)%>"><%=propertyName%>
                         </td>
-                        <input type="hidden" name=<%=name%> id=<%=name%> value="<%=optionals[x].getName()%>"/>
+                        <input type="hidden" name=<%=Encode.forHtmlAttribute(name)%>
+                        id=<%=Encode.forHtmlAttribute(name)%>
```

Comment:
variable name is hardcoded and does not contain special characters. No need to encode that variable